src/easydmp/plan/migrations/0009_auto_20201215_1024.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.13 on 2020-12-15 10:24
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('plan', '0008_auto_20201215_0851'),
```

Comment:
This migration can safely by combined with the "0008_auto_20201215_0851" migration. You could copy the actual migration after the AlterField, or run squash.